src/theme/Admonition/types.css

```diff
@@ -0,0 +1,27 @@
+.custom_admonition {
+  border: #ededed 1px solid;
+  border-left: #6F43E7 solid 4px;
+  border-radius: 10px;
+  margin: 0 0 16px 0;
+  padding: 16px;
+  background-color: #f6f3ff;
```

Comment:
I think we'll need a different set of colors for dark mode, as currently this has poor text contrast:

<img width="828" alt="Screen Shot 2024-01-30 at 12 20 55 PM" src="https://github.com/pomerium/documentation/assets/51246568/6ec7afaf-cfa9-4a7c-a800-2ac9806bae51">